src/components/molecules/ScheduleNG/ScheduleNG.scss

```diff
@@ -0,0 +1,40 @@
+@import "scss/constants.scss";
+
+$ScheduleNG--padding: 20px;
+
+.ScheduleNG {
+  padding-top: $ScheduleNG--padding;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  &__item {
+    display: flex;
+    padding-bottom: $ScheduleNG--padding;
+  }
+
+  &__item--info {
+    flex-basis: 10%;
+    margin-right: $ScheduleNG--padding;
+  }
+
+  &__item--time,
+  &__item--name {
+    display: block;
+    font-weight: 700;
+  }
+
+  &__item--date {
+    display: block;
+    font-weight: 700;
```

Comment:
use variables from `constants.scss`

Reply:
fixed in afad3a98d9284b6bcbe5abf52faa0c6d32e961c4